index.js

```diff
@@ -70,6 +78,83 @@ function MediaSession(opts) {
 }
 
 
+function offerAnswerQueue(self, errorMsg, jingleDesc, cb) {
```

Comment:
nit: the naming of this is confusing to me. Are we queueing an offer/answer? If so I would name it queueOfferAnswer. Otherwise it sounds like it *is* a queue.